bin/iotAgentTester.js

```diff
@@ -60,11 +60,14 @@ function queryContext(commands) {
         },
         headers: {
             'fiware-service': config.service,
-            'fiware-servicepath': config.subservice,
-            'X-Auth-Token': token
```

Comment:
It is expected, see four lines below. What I did in the tester is to give the chance to make unauthenticated as well as authenticated calls. Before this patch, failing in providing a token caused the request to fail, as a null value was given to Node.js http library setHeader() function.